lib/timezone/lookup/geonames.rb

```diff
@@ -14,7 +14,7 @@ def initialize(config)
 
         config.protocol ||= 'http'.freeze
         config.url ||= 'api.geonames.org'.freeze
-
+        config.radius ||= 0
```

Comment:
Won't this set the radius to `0` and pass it in the API request? It might be best to only use this value if it is set.